libs/contentful-translation-extension/README.md

```diff
@@ -0,0 +1,43 @@
+# contentful-translation-extension
+
+This is the Contentful extension responsible of translations for namespaces for your application. It works in pair with the [localization](../localization/README.md) library.
+
+![](https://user-images.githubusercontent.com/937328/105497145-626c4c80-5cb6-11eb-8df8-1b8f19076768.png)
+
+## Usage
+
+Contentful's extensions are using their own bundler and configurations. That's the reason why it's not part of the normal NX workflow.
+
+The following commands are handle inside the `contentful-translation-extension.sh` using contentful's [`contentful-extension-scripts`](https://github.com/jeremybarbet/create-contentful-extension).
+
+{% hint style="warning" %}
+To be able to run the `start` and `deploy` commands you will need to pass an `extensionId`, a `managementToken` and the `path` to the extension's folder.
+{% endhint %}
+
+### Start
+
+Will start the local development server. You are then able to go to Contentful to do your changes.
+
+```bash
+yarn cte start <extensionId> <managementToken> <path>
```

Comment:
How about defining this as an NX workspace, with serve/build/deploy `run-commands` targets? Those targets can set the `extensionId` and `path` values, and pull the `managementToken` from the environment (documented in this readme). It should be fine to use NX, even though this is not using all of their fancy plugins logic.

I'm trying to avoid cluttering the package.json scripts when possible. Maybe we don't need `scripts/contentful-translation-extension.sh` either.

Reply:
Great idea, I will do that!

At first, I started creating a custom schematic and definitely questioned myself if that was worth it, for now, that's why I took this shortcut instead and I didn't think about this nice alternative